cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java

```diff
@@ -499,6 +500,7 @@ private static Injector createInjector(boolean disableAuthorization) throws Exce
       new ProgramRunnerRuntimeModule().getDistributedModules(),
       new SystemDatasetRuntimeModule().getDistributedModules(),
       new NotificationServiceRuntimeModule().getDistributedModules(),
+//      new LoggingModules().getDistributedModules(),
```

Comment:
Why commented?

Reply:
oops - I commented it to make the test fail, to understand why it was needed. Forgot to uncomment it.